CMakeLists.txt

```diff
@@ -9,7 +9,14 @@ set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_VERBOSE_MAKEFILE OFF CACHE BOOL "Enable verbose make")
 set(CMAKE_COLOR_MAKEFILE   ON)
-set( CMAKE_EXPORT_COMPILE_COMMANDS ON )
+
+#Set preprocessor macros depending on used ros version
+if ($ENV{ROS_DISTRO} STREQUAL "foxy")
+  add_definitions(-DROS_FOXY)
+elseif($ENV{ROS_DISTRO} STREQUAL "humble")
+  add_definitions(-DROS_HUMBLE)
+endif()
+
 project(atos VERSION 1.0.0)
```

Comment:
Maybe this is version 1.0.1?

Reply:
Hmmm I wonder if should have version tagging at all here in the CMake file now when we have it on git as well.
I any case we should update the version number automatically here (to be the same as the git tag) instead of hard coding it here.